go/pkg/cas/upload.go

```diff
@@ -265,31 +332,93 @@ type uploader struct {
 
 // startProcessing adds the item to the appropriate stage depending on its type.
 func (u *uploader) startProcessing(ctx context.Context, ps *PathSpec) error {
-	if !filepath.IsAbs(ps.Path) {
-		return errors.Errorf("%q is not absolute", ps.Path)
+	origPsPtr := ps
+
+	// Do not use os.Stat() here. We want to know if it is a symlink.
+	// TODO(nodir): cache this syscall, perhaps using filemetadat package.
+	info, err := os.Lstat(ps.Path)
+	if err != nil {
+		return errors.WithStack(err)
```

Comment:
do you see the linter error after rebase?

Reply:
rebasing will screw up comments, so going to rebase later.

Reply:
done